docs/getting-started/install/linux/index.md

```diff
@@ -12,7 +12,7 @@ Regardless of your distribution, you will need the Mono Project GPG signing key,
 Debian, Ubuntu, and derivatives
```

Comment:
Sounds nice. Any ideas on implementation? Is there existing CSS to handle a horizontal switcher?

Actually, if we go that route, can we punt it to the main Download page?